### PR TITLE
Added isAuditLogged column to object-metadata-entity

### PR DESCRIPTION
**Pull request by Jeetch8**
Added isAuditLogged column to object-metadata-entity.ts

This is my first open source pull request. Please do let me know if made any mistake. I will be greatfull. Thank u

### CODE REVIEW

**Comment by Jeetch8** [comment]
i was trying to close the issue [#4890](https://github.com/twentyhq/twenty/issues/4890)

**Comment by charlesBochet** [comment]
Ok! So I think you also need to modify: EntityEventsToDbListener :)

**Comment by Jeetch8** [comment]
Hii, please check the additions.

File: packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class EntityEventsToDbListener {
     payload: ObjectRecordCreateEvent<any>,
     operation: string,
   ) {
-    if (payload.objectMetadata.isSystem) {
+    if (payload.objectMetadata.isAuditLogged) {
```
Comment (FelixMalfait) [inline packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts:52]:
```suggestion
    if (!payload.objectMetadata.isAuditLogged) {
```

**Comment by FelixMalfait** [comment]
Thanks! You also need to create the migration by running `yarn nx typeorm  twenty-server -- migration:generate src/database/typeorm/metadata/migrations/addIsAuditLogged -d src/database/typeorm/metadata/metadata.datasource.ts` 
I will do it for you

**Review by FelixMalfait — APPROVED** [review]

**Comment by amitshuklabag** [comment]
It still arises today In the docker image, so please fix it in the docker image.

it was arising when I login The login was completed but showed a blank screen and in the network tab API failed with the message 

`extensions:{code: "INTERNAL_SERVER_ERROR", response: "column ObjectMetadataEntity.isAuditLogged does not exist"}
code: "INTERNAL_SERVER_ERROR"
response:"column ObjectMetadataEntity.isAuditLogged does not exist"
message:"QueryFailedError"`

I tried to run migration command as well but same issue `yarn nx typeorm  twenty-server -- migration:generate src/database/typeorm/metadata/migrations/addIsAuditLogged -d src/database/typeorm/metadata/metadata.datasource.ts`

**Comment by amitshuklabag** [comment]
In your new release Now I can log in but have some other issues please check it https://hub.docker.com/layers/twentycrm/twenty/main/images/sha256-51833ef825d15dc7e8c5d64fb2d1c7f8679b31bbc9a783f3d6eb18df7d29e7d5?context=explore multiple API are failed.

**Comment by FelixMalfait** [comment]
Hey @amitshuklabag - the "main" docker image is used for development and is not meant to be deployed. You should use the latest tagged release instead

**Comment by FelixMalfait** [comment]
Arf sorry. It will be fixed in the next release 0.11 for sure, we do extensive checks every 2 weeks when we release new feature

**Comment by amitshuklabag** [comment]
Thanks for updating the latest tag on docker, now the server is on coffee break, issue resolved.

But still, it has issues related to listing the companies, tasks, and activities in the docker latest tag.

**Pull request merged**